src/templates/angular/src/app/pages/header/header.component.html

```diff
@@ -14,6 +14,19 @@
             <a class="nav-link" [routerLink]="'/users'" routerLinkActive="active">User</a>
           </li>
         </ul>
+        <% if(isDark){%>
+          <form class="d-flex" role="button" [formGroup]="headerForm">
+            <div class="menu">
+              <div class="icon">
+                <label class="icon-switcher" data-js-icon-switcher="data-js-icon-switcher" (change)="handleSelection($event)">
+                  <input id="themeSelector" class="control" type="checkbox" formControlName="themeSelection"/>
```

Comment:
themeSelection
is this holds the value of selected theme?
if yes, lets update its name to selectedTheme /optedTheme
selection is a verb can be use for functions not variables